sandy/errorr.py

```diff
@@ -16,36 +16,123 @@ class Errorr(_FormattedFile):
+
+        Returns
+        -------
+        `np.array`
+            The energy grid of the `sandy.Errorr` object.
+
+        Examples
+        --------
+        >>> endf6_2 = sandy.get_endf6_file("jeff_33", "xs", 942410)
+        >>> err = endf6_2.get_errorr(ek=sandy.energy_grids.CASMO12, err=1)
+        >>> err.get_energy_grid()
+        array([1.0000e-05, 3.0000e-02, 5.8000e-02, 1.4000e-01, 2.8000e-01,
+               3.5000e-01, 6.2500e-01, 4.0000e+00, 4.8052e+01, 5.5300e+03,
+               8.2100e+05, 2.2310e+06, 1.0000e+07])
+        """
+        mat_ = mat if mat else self.mat[0]
+        mf1 = read_mf1(self, mat_)
+        return mf1["EG"]
 
     def get_xs(self, mat, mt):
         """
-        
+        Obtain for a given mat and mt the xs values across the energy grid.
 
         Parameters
         ----------
-        mat : TYPE
-            DESCRIPTION.
-        mt : TYPE
-            DESCRIPTION.
+        mat : `int`
+            MAT number
+        mt : `int`
+            MT number
 
         Returns
         -------
-        xs : TYPE
-            DESCRIPTION.
-
+        xs : `pd.Series`
+            For a given mat and mt, the xs values in the energy grid.
+
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "xs", 10010)
+        >>> err = endf6.get_errorr(ek=sandy.energy_grids.CASMO12, err=1)
+        >>> err.get_xs(125, 102).head()
+        (1e-05, 0.03]   6.17622e-01
+        (0.03, 0.058]   2.62307e-01
+        (0.058, 0.14]   1.77108e-01
+        (0.14, 0.28]    1.21068e-01
+        (0.28, 0.35]    1.01449e-01
+        Name: (125, 102), dtype: float64
         """
         mf1 = read_mf1(self, mat)
         mf3 = read_mf3(self, mat, mt)
         index = pd.IntervalIndex.from_breaks(mf1["EG"])
         xs = pd.Series(mf3["XS"], index=index, name=(mat, mt))
         return xs
+    
+    def get_cov(self):
+        """
+        Extract cross section/nubar covariance from `Errorr` instance. 
+
+        Returns
+        -------
+        data : `sandy CategoryCov`
+            xs/nubar covariance matrix for all cross section/nubar
+            MAT/MT in ERRORR file.
+
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "xs", 10010)
+        >>> err = endf6.get_errorr(ek=sandy.energy_grids.CASMO12, err=1)
+        >>> err.get_cov().data.iloc[:5, :5]
+                MAT1	125
+                MT1	1
+                E1	1.00000e-05	3.00000e-02	5.80000e-02	1.40000e-01	2.80000e-01
+        MAT	MT	E					
+        125	1	1.00000e-05	8.82899e-06	8.65234e-06	8.60908e-06	8.58043e-06	8.57036e-06
+                3.00000e-02	8.65234e-06	8.65909e-06	8.66074e-06	8.66183e-06	8.66221e-06
+                5.80000e-02	8.60908e-06	8.66074e-06	8.67339e-06	8.68176e-06	8.68471e-06
+                1.40000e-01	8.58043e-06	8.66183e-06	8.68176e-06	8.69497e-06	8.69961e-06
+                2.80000e-01	8.57036e-06	8.66221e-06	8.68471e-06	8.69961e-06	8.70484e-06
```

Comment:
can you add an example for H1 using a 2-group energy structure? Then we can see the entire matrix